projects/packages/connection-ui/composer.json

```
@@ -0,0 +1,22 @@
+{
+	"name": "automattic/jetpack-connection-ui",
+	"description": "Jetpack Connection UI",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {
+		"automattic/jetpack-connection": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../*"
+		}
+	],
+	"minimum-stability": "dev",
```

```suggestion
	"scripts": {
		"build-development": [
			"Composer\\Config::disableProcessTimeout",
			"yarn build"
		]
	},
	"minimum-stability": "dev",
```

There's a new `jetpack build` command that just merged that would need this to build the package centrally.

---

Done, thanks!